### PR TITLE
Brooklyn properties files may be symlinks

### PR DESCRIPTION
**Pull request by sjcorbett**
The `?00` permissions check is enforced on the link's target rather than the link itself.

### CODE REVIEW

**Comment by sjcorbett** [comment]
`Build timed out (after 40 minutes). Marking the build as aborted.`

File: usage/launcher/src/main/java/org/apache/brooklyn/launcher/BrooklynLauncher.java [hunk 1]
```diff
@@ -710,32 +719,42 @@ private void initManagementContext() {
...
+     */
+    private File resolveSymbolicLink(File f) {
+        File f2 = f;
+        try {
+            f2 = f.getCanonicalFile();
+            if (Files.isSymbolicLink(f.toPath())) {
+                LOG.debug("Resolved symbolic link: {} -> {}", f, f2);
+            }
+        } catch (IOException e) {
+            LOG.warn("Could not determine canonical name of global properties file", e);
```
Comment (aledsage) [inline usage/launcher/src/main/java/org/apache/brooklyn/launcher/BrooklynLauncher.java]: Perhaps change log message to not say `global properties file`, as this method is more generic (even if currently only called in that one case)?

How about `Could not determine canonical path of file "+f, e`

**Comment by aledsage** [comment]
@sjcorbett Looks good - one very minor comment about an improvement to a log message. Good to merge.

**Comment by ahgittin** [comment]
agree re log message, i've applied that and merged

**Pull request merged**